src/utils/tezos.js

```diff
@@ -65,6 +65,10 @@ const getBatch = () => {
   return Tezos.wallet.batch([]);
 };
 
+const getTezosBatch = (operations) => {
+  return Tezos.wallet.batch(operations);
+}
+
```

Comment:
we already have `getBatch`, just change it to take in an operations parameter with the default being an empty array.

Reply:
Reuse getBatch